### ISSUE TITLE
Exporting to APNG fails in master branch

### ISSUE THREAD

**Issue by sindresorhus** [wulkano/kap#538]
Master meaning https://github.com/wulkano/kap/commit/285a26dac425a35ab759ef76b629584e6a5fb4a6

I just get "Export failed" in the Exports window. No error message in devtools.

When fixing this we should ensure that if it fails, at least it shows a more detailed error message in devtools.

**Comment by skllcrn** [comment]
I realise that you're referring to `master`, but just to be clear I'm not able to reproduce this in the latest beta. Wonder what could be causing it?

**Comment by karaggeorge** [comment]
I just tested this in the current master and it exported APNG successfully. I'll open a PR for showing any export errors in the devtools however, so we can figure out what it was if it's something that doesn't happen often

**Comment by sindresorhus** [comment]
Thanks to #547, here's the error I'm getting:

```
index.js:2178 Error: ffmpeg exited with code: 1

ffmpeg version N-91158-g63c4a4b0d6-tessus Copyright (c) 2000-2018 the FFmpeg developers
  built with Apple LLVM version 9.1.0 (clang-902.0.39.1)
  configuration: --cc=/usr/bin/clang --prefix=/opt/ffmpeg --extra-version=tessus --enable-avisynth --enable-fontconfig --enable-gpl --enable-libaom --enable-libass --enable-libbluray --enable-libfreetype --enable-libgsm --enable-libmodplug --enable-libmp3lame --enable-libmysofa --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopus --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvo-amrwbenc --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libx264 --enable-libx265 --enable-libxavs --enable-libxvid --enable-libzimg --enable-libzmq --enable-libzvbi --enable-version3 --pkg-config-flags=--static --disable-ffplay
  libavutil      56. 18.102 / 56. 18.102
  libavcodec     58. 19.104 / 58. 19.104
  libavformat    58. 17.100 / 58. 17.100
  libavdevice    58.  4.100 / 58.  4.100
  libavfilter     7. 24.100 /  7. 24.100
  libswscale      5.  2.100 /  5.  2.100
  libswresample   3.  2.100 /  3.  2.100
  libpostproc    55.  2.100 / 55.  2.100
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from '/private/var/folders/3x/jf5977fn79jbglr7rk0tq4d00000gn/T/7f201309383941ff8c8f380c3a008e00.mp4':
  Metadata:
    major_brand     : qt  
    minor_version   : 0
    compatible_brands: qt  
    creation_time   : 2018-09-28T02:40:09.000000Z
    com.apple.quicktime.make: Apple
    com.apple.quicktime.model: MacBookPro11,3
    com.apple.quicktime.software: Mac OS X 10.14 (18A391)
    com.apple.quicktime.creationdate: 2018-09-28T09:40:08+0700
  Duration: 00:00:06.77, start: 0.000000, bitrate: 1635 kb/s
    Stream #0:0(und): Video: h264 (Main) (avc1 / 0x31637661), yuv420p(tv, bt709), 1206x652 [SAR 1:1 DAR 603:326], 1210 kb/s, 30 fps, 30 tbr, 6k tbn, 12k tbc (default)
    Metadata:
      creation_time   : 2018-09-28T02:40:09.000000Z
      handler_name    : Core Media Data Handler
      encoder         : H.264
    Stream #0:1(und): Audio: aac (LC) (mp4a / 0x6134706D), 44100 Hz, stereo, fltp, 145 kb/s (default)
    Metadata:
      creation_time   : 2018-09-28T02:40:09.000000Z
      handler_name    : Core Media Data Handler
Stream mapping:
  Stream #0:0 -> #0:0 (h264 (native) -> apng (native))
Press [q] to stop, [?] for help
[APNG muxer @ 0x7fe1d85004c0] Value -1.000000 for parameter 'plays' out of range [0 - 4.29497e+09]
[APNG muxer @ 0x7fe1d85004c0] Error setting option plays to value -1.
Could not write header for output file #0 (incorrect codec parameters ?): Result too large
Error initializing output stream 0:0 -- 
Conversion failed!

    at ChildProcess.converter.on.code (Users/sindresorhus/d…/main/convert.js:59)
    at ChildProcess.emit (events.js:187)
    at ChildProcess.EventEmitter.emit (domain.js:442)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:237)
```

**Comment by karaggeorge** [comment]
Much better 👍 I'll see what's going on with that as well

**Comment by sindresorhus** [comment]
Fixed by #547
